server/src/main/java/org/apache/iotdb/db/qp/logical/Operator.java

```diff
@@ -192,6 +192,8 @@ public PhysicalPlan generatePhysicalPlan(PhysicalGenerator generator)
     UNSET_TEMPLATE,
     PRUNE_TEMPLATE,
     APPEND_TEMPLATE,
-    DROP_TEMPLATE
+    DROP_TEMPLATE,
+
```

Comment:
Reduce unnecessary blank lines

Reply:
there is another type Operator behind, so a blank is here to separate them~